test/system/065-cp.bats

```diff
@@ -110,8 +149,23 @@ load helpers
         is "$output" "${randomcontent[$id]}" "$description (cp ctr:$src to \$srcdir$dest)"
         rm $srcdir/$dest_fullname
     done < <(parse_table "$tests")
+    run_podman rm -f cpcontainer
 
+    # Created container
+    run_podman create --name cpcontainer --workdir=/srv $cpimage
+    while read id src dest dest_fullname description; do
+        # dest may be "''" for empty table cells
+        if [[ $dest == "''" ]];then
+            unset dest
+        fi
+        run_podman cp cpcontainer:$src "$srcdir$dest"
+        run cat $srcdir$dest_fullname
+        is "$output" "${randomcontent[$id]}" "$description (cp ctr:$src to \$srcdir$dest)"
```

Comment:
Hint:
```bash
    (remove the "run cat")
    is "$(< $srcdif$dest_fullname)" "${random...."
```
`$(< PATH)` is an idiomatic equivalent to `$(cat PATH)`

Reply:
Nice!  I'll get that consistent across the copy tests.